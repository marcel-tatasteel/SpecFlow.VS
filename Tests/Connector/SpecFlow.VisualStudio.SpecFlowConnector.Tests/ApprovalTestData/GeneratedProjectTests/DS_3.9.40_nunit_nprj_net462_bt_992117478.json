{
  "GeneratorOptions": {
    "FeatureFileCount": 1,
    "ScenarioPerFeatureFileCount": 1,
    "NewProjectFormat": true,
    "TargetFramework": "net462",
    "SpecFlowPackageVersion": "3.9.40"
  }
}
