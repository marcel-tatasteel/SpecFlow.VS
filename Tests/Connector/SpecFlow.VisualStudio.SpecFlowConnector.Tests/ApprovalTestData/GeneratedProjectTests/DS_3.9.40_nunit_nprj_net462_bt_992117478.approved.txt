{
  "stepDefinitions": [],
  "sourceFiles": {},
  "typeNames": {},
  "analyticsProperties": {
    "Connector": "<connector>",
    "ImageRuntimeVersion": "v4.0.30319",
    "SFFile": "TechTalk.SpecFlow.dll",
    "SFFileVersion": "3.9.40.46609",
    "SFProductVersion": "3.9.40+11b6bbec89",
    "SourcePaths": "1",
    "StepDefinitions": "3",
    "TargetFramework": ".NETFramework,Version=v4.6.2",
    "TypeNames": "0"
  }
}